lib/galaxy/schema/schema.py

```diff
@@ -999,6 +999,11 @@ class CreateNewCollectionPayload(Model):
         default=None,
         description="The ID of the history that will contain the collection. Required if `instance_type=library`.",
     )
+    fields: Optional[Union[str, List[Dict['str', 'str']]]] = Field(
+        default=[],
+        description="List of fields to create for this collection. Set to 'auto' to guess fields from identifiers.",
+        alias="fields_",
+    )
 
```

Comment:
I think what you need here is the opposite, so there is no conflict with the internal `fields` property of `BaseModel`.
```suggestion
    fields_: Optional[Union[str, List[Dict['str', 'str']]]] = Field(
        default=[],
        description="List of fields to create for this collection. Set to 'auto' to guess fields from identifiers.",
        alias="fields",
    )

```